unlock-app/src/pages/demo.js

```diff
@@ -100,9 +112,77 @@ const mapStateToProps = state => {
 
 export default withConfig(connect(mapStateToProps)(Demo))
 
-const BottomSticker = styled.p`
-  overflow-wrap: break-word;
-  word-break: break-all;
-  background-color: var(--lightgrey);
-  padding: 10px 30px;
+const GlobalStyle = createGlobalStyle`
+    @import url('https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,700|UnifrakturCook:700');
+
+    body {
+      background - color: #fdfaf7;
+      font - family: 'Source Serif Pro', serif;
+    }
+    `
+
+const Container = styled.div`
+  display: grid;
+  max-width: 1400px;
+  grid-template - columns: 1fr minmax(500px, 3fr) 1fr;
+
+  @media (max - width: 500px) {
+    grid-template - columns: 0px 1fr 0px;
+  }
+`
```

Comment:
```suggestion
`

```

I feel like we should try for consistent spacing between the styled components here - although this is the nittiest of tiny nits, I know!